code/core/src/manager/components/Particles/Particles.tsx

```diff
@@ -0,0 +1,165 @@
+    <path d="M28 3v25a3 3 0 003 3h22a3 3 0 013 3v53a3 3 0 01-3 3H31a3 3 0 01-3-3V62a3 3 0 00-3-3H3a3 3 0 01-3-3V3a3 3 0 013-3h22a3 3 0 013 3z" />
+  </Shape>
+);
+
+const fadeToTransparent = keyframes`
+  to {
+    opacity: 0;
+  }
+`;
+
+const disperse = keyframes`
+  to {
+    transform: translate(
+      calc(cos(var(--angle)) * var(--distance)),
+      calc(sin(var(--angle)) * var(--distance))
+    ) rotate(var(--rotation));
+  }
+`;
+
+const slideDown = keyframes`
+  to {
+    transform: translateY(50px);
+  }
+`;
+
+const Container = styled.div({
+  position: 'absolute',
+  display: 'flex',
+  flexDirection: 'row',
+  flexWrap: 'wrap',
+  gap: 10,
+  '--particle-curve': 'cubic-bezier(0.2, 0.56, 0, 1)',
+  animation: `${slideDown} 1000ms forwards cubic-bezier(0.587, -0.381, 0.583, 0.599)`,
+  animationDelay: '150ms',
+  zIndex: 999,
+
+  svg: {
+    width: 15,
+    height: 15,
+    animation: `${fadeToTransparent} var(--fade-duration) forwards, ${disperse} 1000ms forwards var(--particle-curve)`,
+  },
+});
+
+const FADE_DURATION = 1200;
+const NUM_OF_PARTICLES = 8;
+// `JITTER` is the amount of variance allowed for each angle.
+// Tweak this value to control how orderly/chaotic the animation appears.
+const JITTER = 15;
+
+const random = (min: number, max: number) => Math.random() * (max - min) + min;
+const sortRandomly = (array: any[]) => array.sort(() => Math.random() - 0.5);
+
+export const Particles = memo(function Particles({
+  anchor: Anchor,
+}: {
+  anchor: React.ElementType;
+}) {
+  const anchorRef = useRef<HTMLDivElement>(null);
+  const [left, setLeft] = useState(0);
+  const [top, setTop] = useState(0);
+
+  const shapes = sortRandomly([Donut, L, Slice, Square, Triangle, T, Z]);
+  const colors = sortRandomly([
+    '#FF0000',
+    '#FF4787',
+    '#66BF3C',
+    '#1EA7FD',
+    '#FC521F',
+    '#6F2CAC',
+    '#FFAE00',
+  ]);
+
+  useLayoutEffect(() => {
+    const rect = anchorRef.current?.getBoundingClientRect();
+    if (rect) {
+      setLeft(rect.left + rect.width / 2);
+      setTop(rect.top + rect.height / 2);
+    }
+  }, []);
+
+  return (
+    <div ref={anchorRef}>
+      <Anchor />
+      {createPortal(
+        <Container style={{ top: top + 'px', left: left + 'px' }}>
+          {shapes.map((Particle, index) => {
+            const angle = (360 / NUM_OF_PARTICLES) * index + random(-JITTER, JITTER);
+            const distance = random(50, 80);
+            const rotation = random(-360, 360);
+
+            const style = {
+              '--angle': angle + 'deg',
+              '--distance': distance + 'px',
+              '--rotation': rotation + 'deg',
+              '--fade-duration': FADE_DURATION + 'ms',
+            } as React.CSSProperties;
+
+            return <Particle key={Particle.name} style={style} color={colors[index]} />;
+          })}
+        </Container>,
+        document.getElementById('root') ?? document.body
```

Comment:
Can you explain this:
```ts
document.getElementById('root')
```
Why would this ever happen?
Should this be in the implementation-code if it's for testing/stories, how else should we solve this.

I suspect this could cause unexpected behavior down the line.